src/index.html

```diff
@@ -13,10 +13,93 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
```

Comment:
The `href` attribute for the navigation links is set to `#`, which is not ideal for accessibility and usability. Consider using actual URLs or JavaScript to handle navigation if these are placeholders.